exercises/circumference-of-a-circle/circumference-of-a-circle.js

```diff
@@ -0,0 +1,14 @@
+function main() {
+    var r = parseInt(readLine(), 10)
+    
+    //the output
+    console.log(calcCirc(r).toFixed(2));
```

Comment:
Every time you use the function calcCirc() you'll need to do the `.toFixed(2)`.
To prevent this, you can move this code inside the `calcCirc()` function.

Reply:
So, that would be something like this?
```
function calcCirc(r){
    return(2 * Math.PI * r).toFixed(2);
}
```
Can I do this return().toFixed()?

Reply:
Yes, you can.
If you want, you can improve the code a lot by adding this features:
- Validate input `r` before using it. For example, validate if `r` is greater than 0
- First do the `2* Math.PI * r`, then verify if the value mod 2 is different than 0. Then you can apply the `toFixed(2)` only if the mod 2 returns a value different than 0 - this way, you improve the code performance.

Reply:
I did new commit to the branch with my attempt 😄